config/initializers/devise_token_auth.rb

```diff
@@ -19,7 +19,7 @@
   # time. In this case, each request in the batch will need to share the same
   # auth token. This setting determines how far apart the requests can be while
   # still using the same auth token.
-  # config.batch_request_buffer_throttle = 5.seconds
+  # config.batch_request_buffer_throttle = 1.day
```

Comment:
Why?

Reply:
part of the big problen with autentication, it sets the time between new tokens of user, it is just to pass the tests, we had a really big problem with that.